tools/check-fenced-code.mjs

```diff
@@ -0,0 +1,65 @@
+import { promisify } from 'util';
+import fs from 'fs-extra';
+import g from 'glob';
+import MarkdownIt from 'markdown-it';
+import shell from 'shelljs';
+
+const glob = promisify(g);
+
+const errorTitle = 'Invalid JSON in fenced code block';
+const errorBody =
+  'Fix this manually by ensuring each block is a valid, complete JSON document';
```

Comment:
```suggestion
  'Fix this manually by ensuring each block is a valid, complete JSON document.';
```

Add missing full stop at end of sentence.

Reply:
Thanks @HonkingGoose I reshuffled this a bit :)